src/main/java/net/mcreator/ui/component/util/DiscordClient.java

```diff
@@ -96,4 +120,7 @@ public void updatePresence(String state, String details, String smallImage) {
 			LOG.warn("Failed to close properly", e);
 		}
 	}
+	private boolean isDisabled() {

```

Comment:
There should be one line above and below the method to match style of neighboring code